client/my-sites/domains/domain-management/gsuite/gsuite-purchase-cta/index.jsx

```diff
@@ -15,79 +15,81 @@ import React, { Fragment } from 'react';
  */
 import { ADDING_GOOGLE_APPS_TO_YOUR_SITE } from 'lib/url/support';
 import Button from 'components/forms/form-button';
-import { canAddGoogleApps, getGoogleAppsSupportedDomains } from 'lib/domains';
 import CompactCard from 'components/card/compact';
-import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
 import config from 'config';
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
 import { domainManagementAddGoogleApps } from 'my-sites/domains/paths';
+import EmailVerificationGate from 'components/email-verification/email-verification-gate';
 import { getAnnualPrice, getMonthlyPrice } from 'lib/google-apps';
 import { getCurrentUserCurrencyCode } from 'state/current-user/selectors';
 import { getDomainsBySiteId } from 'state/sites/domains/selectors';
+import { getEligibleDomain } from 'lib/domains/gsuite';
 
-class AddGoogleAppsCard extends React.Component {
-	constructor( props ) {
-		super( props );
-	}
-
-	renderAddGoogleAppsButton() {
-		const { translate } = this.props;
-
-		if ( ! config.isEnabled( 'upgrades/checkout' ) ) {
-			return null;
-		}
-
-		return (
-			<Button type="button" onClick={ this.goToAddGoogleApps }>
-				{ translate( 'Add G Suite' ) }
-			</Button>
-		);
-	}
+/**
+ * Style dependencies
+ */
+import './style.scss';
 
+class GSuitePurchaseCta extends React.Component {
 	handleLearnMoreClick = () => {
-		this.props.learnMoreClick( this.props.eligibleDomainName );
+		this.props.learnMoreClick( this.props.domainName );
 	};
 
 	goToAddGoogleApps = () => {
-		page(
-			domainManagementAddGoogleApps( this.props.selectedSite.slug, this.props.eligibleDomainName )
-		);
+		page( domainManagementAddGoogleApps( this.props.selectedSite.slug, this.props.domainName ) );
 	};
 
-	render() {
-		const { currencyCode, eligibleDomainName, translate } = this.props,
-			price = get( this.props, [ 'products', 'gapps', 'prices', currencyCode ], 0 ),
-			googleAppsSupportUrl = ADDING_GOOGLE_APPS_TO_YOUR_SITE;
+	getStorageText() {
+		const { product, translate } = this.props;
+		if ( 'gapps' === product.product_slug ) {
+			return translate( 'Get 30GB of storage for all your files synced across devices.' );
+		} else if ( 'gappsbusiness' === product.product_slug ) {
+			return translate( 'Get 30GB of storage for all your files synced across devices.' );
```

Comment:
@belcherj: we use the same message in both cases, is that expected? Or should me merge the `if` conditions?